src/components/UI/Footer/Footer.tsx

```diff
@@ -39,19 +31,22 @@ const FooterColumn = (props: FooterColumnProps): React.ReactElement => {
 
 export const Footer = (props: FooterProps): React.ReactElement => {
   const { className = "", title = "", logoUrl = "", copyright, data } = props;
+  const topData = data ? data.filter((data) => data.category.includes("Category")) : [];
+  const bottomData = data ? data.filter((data) => data.category.includes("Bottom")) : [];
```

Comment:
dont reuse data. rename data appropriately and then Introduce a new prop for the new data and pass it in instead

Reply:
ok updated